### ISSUE TITLE
Mouse support

### ISSUE THREAD

**Issue by AndiLeni** [lhmouse/nano-win#10]
Hello,

I enabled mouse support int the .nanorc file, but it is not working for me inside the console.
Is this behavior intended or is this a bug?

Regards
Andi

**Comment by lhmouse** [comment]
If you have **Quick edit mode** enabled in your console properties you must have disabled.

**Comment by AndiLeni** [comment]
Yes, I have Quick Edit mode enabled in my console, but if I disable it in my console and enable it in the nanorc file, it is still not working, then the mouse support is not working at all.

**Comment by lhmouse** [comment]
1. If you are on **Windows 10** try enabling the legacy console. Mouse works for me on **Windows 7**.
2. You may switching mouse support on and off using **Alt-M**. This also helps you identify whether it is really enabled.

**Comment by eabase** [comment]
This is not working in W10. Cannot get mouse to work, regardless of settings above. 
Also seem that nano doesn't remember last cursor location in file.
